tests/Integration/TestCase.php

```diff
@@ -51,6 +56,26 @@ public function setUp($defaultSerializer = '')
                 'publisher' => 'Bloody Fantasy inc.',
             ],
         ];
+
+	    $this->testPosts = [
```

Comment:
The secound post is because i wanted to test the exclude functionallity. With the same book and just a new transformer. 
I'll change it back no problem.